.github/next_release_version.bash

```diff
@@ -0,0 +1,53 @@
+#! /bin/bash
+
+# Defaults
+next_version_release_year=$(TZ='UTC' date '+%Y')
+next_version_release_number=1
+
+if [[ $1 == 'test' ]]; then
+    echo 'Running tests...' >&2
+    dotest() {
+        result=$(bash $0 "release/${1}" 2> /dev/null | tail -n 1)
+        expect="${2}"
+        if [[ $result != $expect ]]; then
+            printf "Test failed:\n  Expected: $expect\n  Received: $result\n" >&2
+            exit 1
+        fi
+    }
+    dotest 'release/1234.987' "$next_version_release_year.1"
```

Comment:
why use date based versioning over semantic versioning?

Reply:
Per [this note](https://github.com/usdigitalresponse/usdr-gost/blob/409dc50dbbfc6c663b7a3c289ce4e4b1faf988d7/docs/releasing.md#why-not-semantic-versioning) in the newly-added `docs/releasing.md` file:
> Why not semantic versioning?
> 
> Many software projects (especially libraries) follow versioning schemes like [semantic versioning](https://semver.org/) because it standardizes an explicit way of communicating compatibility changes from one version to the next. You probably recognize semantic versions, like `v1.2.3`. However, because the usdsr-gost repository provides multiple services and offers no single, concrete contract (i.e. an API) about which downstream consumers contend with compatibility decisions, a semantic versioning scheme is less relevant and would likely create ambiguilties rather than clarifying them.

The honest answer here is that I don't have a great conception of what a "breaking" vs "backwards-compatible" change would really look like for this repo (and I've generally struggled to conceptualize for any service more complex than a single API). Given that this repository houses the client and server codebases for two mostly (but not completely) separate fullstack web applications, I feel like it would be difficult to intuitively decide on a semantic version number that wasn't just a constant minor-version increment. Without any external dependencies to consider, it seemed like the clearest way to approach versioning would simply be to defer to the release date of the changeset.

Reply:
That's fair - mono repos can have that problem